client/state/audio/README.md

```diff
@@ -0,0 +1,36 @@
+# Audio Middleware
+
+## Overview
+
+This middleware plays audio when certain Redux actions are dispatched. This can be useful for adding
+audio feedback when some event occurs in Calypso.
+
+For example, the initial use of this is to play a notification sound when a new message is received in Happychat.
+
+Only browsers supporting the [`new Audio()` API](https://developer.mozilla.org/en/docs/Web/API/HTMLAudioElement)
+will play a sound, other browsers will ignore this middleware. Currently works on Chrome, Edge,
+Firefox, and Safari.
+
+## Playing a sound
+
+To play a sound when an action is dispatched, add a new handler in `./middleware.js`:
+
+For example, if a sound should be played when an action of type `SOME_ACTION_TYPE` is dispatched, add the
+following:
+
+```js
+export const onSomeEvent = ( dispatch, action, getState ) => {
+	playSound( '/calypso/audio/sound-to-play.wav' );
+};
+
+// ...
+
+/**
+ * Action Handlers
+ */
+
+// ...
+
+handlers[ SOME_ACTION_TYPE ] = onSomeEvent;
+
+```
```

Comment:
this code snippet is a bit confusing to me. do you think we could get a little more context here? maybe add two example events, one of which performs some logical test, one that doesn't? I think adding the HappyChat function could be a good example.

how about also putting the middleware filename as a comment at the top of the example block? help us orient ourselves in the middleware